internal/core/api.rs

```diff
@@ -383,23 +383,42 @@ impl Window {
 
 pub use crate::input::PointerEventButton;
 
-/// An event sent to a window
+/// A event that describes user input.
+///
+/// Slint backends typically receive events from the windowing system, translate them to this
+/// enum and deliver to the scene of items via [`Window::dispatch_event()`].
+///
+/// The pointer variants describe events originating from an input device such as a mouse
+/// or a contact point on a touch-enabled surface.
+///
+/// All position fields are in logical window coordinates.
 #[allow(missing_docs)]
```

Comment:
maybe not needed anymore?

Reply:
I just checked, it's still needed :( - unless we duplicate the position fields. I'll keep it for now.